libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -201,6 +205,9 @@ BlockCoordinator::BlockCoordinator(MainChain &chain, DAGPtr dag, StakeManagerPtr
     }
   });
 
+  // Initialising the BLS library
+  crypto::bls::Init();
```

Comment:
Where should this go, in the beacon service?

Reply:
Yeah this is annoying, maybe we can simply update the BLS wrapper library to always call this before an operation?

Reply:
In the constructor of beacon service would make sense since I would think but yeah don't really care - I was going to let Troels put it where he likes